### ISSUE TITLE
Could not import project: Missing package/tool-cmake

### ISSUE THREAD

**Issue by KenWood5020** [platformio/platformio-home#1763]
### Configuration
**Operating system:** Linux-5.4.72-v7l+-armv7l-with-glibc2.28
**PlatformIO Version:** Core 5.0.3 Home 3.3.1

### Description of problem
When I try to open an example or create a new project, I get an error message indicating that package tool-cmake was missing (see below). Same message appears on compile.

``` PIO Core Call Error: "The current working directory /home/pi/Documents/PlatformIO/Projects/201125-195252-espidf-arduino-blink will be used for the project.\n\nThe next files/directories have been created in /home/pi/Documents/PlatformIO/Projects/201125-195252-espidf-arduino-blink\ninclude - Put project header files here\nlib - Put here project specific (private) libraries\nsrc - Put project source files here\nplatformio.ini - Project Configuration File\ngit version 2.20.1\n\n\nKlone nach '/home/pi/.platformio/.cache/tmp/pkg-installing-81op7is0' ...\nChecke Dateien aus:  92% (1844/1988)   \rChecke Dateien aus:  93% (1849/1988)   \rChecke Dateien aus:  94% (1869/1988)   \rChecke Dateien aus:  94% (1877/1988)   \rChecke Dateien aus:  94% (1887/1988)   \rChecke Dateien aus:  95% (1889/1988)   \rChecke Dateien aus:  95% (1895/1988)   \rChecke Dateien aus:  96% (1909/1988)   \rChecke Dateien aus:  96% (1915/1988)   \rChecke Dateien aus:  97% (1929/1988)   \rChecke Dateien aus:  98% (1949/1988)   \rChecke Dateien aus:  99% (1969/1988)   \rChecke Dateien aus: 100% (1988/1988)   \rChecke Dateien aus: 100% (1988/1988), Fertig.\nError: Could not find the package with 'platformio/tool-cmake @ ~3.16.0' requirements for your system 'linux_armv7l'" ```

**Steps to Reproduce**
1. Select "Project Example"
2. Select espidf-blink

or

1. Select "New Project"
2. Type in project name, select "Espressif ESP32 Dev module" as board and "Espressif  IoT Development Framework"
3. Check "Finish"

**Build message:**
``` > Executing task: pio run --environment esp32dev <

Processing esp32dev (platform: https://github.com/platformio/platform-espressif32.git; board: esp32dev; framework: espidf)
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Tool Manager: Installing platformio/tool-cmake @ ~3.16.0
Error: Could not find the package with 'platformio/tool-cmake @ ~3.16.0' requirements for your system 'linux_armv7l'
The terminal process terminated with exit code: 1
```

### Actual Results
Platformio aborts project creation and shows above mentioned error in pop up.

### Expected Results
Platformio creates a projects

### If problems with PlatformIO Build System:
**The content of ```platformio.ini:```**
``` [env:esp32dev]
platform = https://github.com/platformio/platform-espressif32.git
board = esp32dev
framework = espidf 
```

**Comment by KenWood5020** [comment]
I was trying to download the missing packages from https://bintray.com/platformio/dl-packages
as described in https://docs.platformio.org/en/latest/faq.html#package-manager, but I only get a Error 404 page.
Can somebody please give me the correct link for the packages?

**Comment by KenWood5020** [comment]
ok, mad some major changes on my system:
1 Switched from Rasbian OS 32 bit to 64 bit
2. Changed from Code OSS to VSC

After spening hours with a similar error (missing toolchain-xtensa32 due to not available version [2.80200.0]) I made a little step further, by just editing  platforms/espressif32/platform.json:
Changed requires version manually to 2.50200.0
Next problem: Missing toolchain-esp32ulp
I downloaded the files from https://bintray.com/platformio/tool-packages/toolchain-esp32ulp-linux_armv6l#files
and put them in packages/toolchain-esp32ulp, added linux, aarch64 to the package.json, but pio does not recognize them.
```Error: Could not find the package with 'platformio/toolchain-esp32ulp @ ~1.22851.0' requirements for your system 'linux_aarch64'```

But this is a different problem. Closing this ticket.
